app-backend/project/Dependencies.scala

```diff
@@ -33,7 +33,7 @@ object Version {
   val geotools = "17.1"
   val geotrellisContrib = "3.16.1-M3"
   val geotrellis = "3.0.0-M3"
-  val geotrellisServer = "3.4.0"
+  val geotrellisServer = "3.4.0-6-ge843f05-SNAPSHOT"
```

Comment:
Are we able to release with this? I don't think we can ship things with snapshot dependencies to Maven central. @notthatbreezy is that right?

Reply:
Ah - yeah - it's [this](https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/build.sbt#L156) line I think that is going to cause that to error. Not sure what the best option for us is in this case.

We could remove the requirement - we could also keep it, but if this is still around when we want to do another release the publish step will fail.